subscription-service/src/test/kotlin/com/egm/stellio/subscription/utils/ParsingUtilsTests.kt

```diff
@@ -70,4 +71,29 @@ class ParsingUtilsTests {
             })
         }
     }
+
+    @Test
+    fun `it should not allow a subscription if remote JSON-LD @context cannot be retrieved`() {
+        val contextNonExisting = "https://uri.etsi.org/ngsi-ld/v1/ngsi-ld-core-context-non-existing.jsonld"
+        val subscription = mapOf(
+            "id" to "invalidId",
```

Comment:
this `invalidId` is a bit misleading (one can wonder if it has a link with the test result). better use a valid one to make things clear.